source/documentation/using_the_api/register_components.md

```diff
@@ -1,18 +1,30 @@
-## The relationship between entries, items and records
+## The components of a register
 
-Registers are made up of:
+Registers are primarily composed of:
 
 * entries
 * items
 * records
 
+### Entries
+
 Entries contain metadata about what and when data has changed. 
 
-Each entry in a register has a relationship with one item, which contains data. An item is connected to an entry with an item hash.   
+Entries are made up of the fields `entry-number`, `index-entry-number`, `entry-timestamp`, `item-hash` and `key`. 
+
+The `entry-number` is unique and defines an entry’s position within the ordered list of a register. The `index-entry-number` is unique and defines an entry’s position within the ordered list of an index. For any given entry in a register, the `entry-number` and `index-entry-number` are always identical. 
```

Comment:
@michaelabenyohai given that indexes are not a stable feature I'm thinking that the description of `index-entry-number` should be separated from the `entry-number` with the note/warning that it is not ready to be used by regular users.

Reply:
Makes sense to me. We should include a definition though, as it's in all the [example] outputs.

Reply:
Sure, as long as it is clear that no one should rely on it. Also, the concept of "index" is not used at the user level right now so mentioning here is not that useful, I think.

Reply:
Hmm, perhaps a separate PR then?